plugins/jspsych-form.js

```diff
@@ -180,10 +180,10 @@ var schema = {
           trial_data[key] = value;
         }
 
-        if (customized_output) 
+        if (customized_output)
           trial_data["#Customized Output#"] = customized_output;
-
-        display_element.html('');
+        // We have to find what's going on here...
+        $(display_element).html('');
```

Comment:
This won't work because $ is not defined by default in jsPsych since the removal of jQuery as a dependency.